tests/Fixtures/inc/Engine/Optimization/DelayJs/Admin/Settings/restoreDefaults.php

```diff
@@ -1,21 +1,11 @@
 <?php
 return [
-	'shouldRestoreDefaults' => [
-		'input' => [
-			'capabilities' => [ 'rocket_manage_options' ],
-			'options' => [
-				'delay_js_scripts' => []
-			]
-		],
-		'restored' => true
+	'testShouldNotRestoreDefaultsWhenNoCapabilities' => [
+		'capability' => false,
+		'restored'   => false,
+	],
+	'testShouldRestoreDefaultsWhenCapabilities'      => [
+		'capability' => true,
+		'restored'   => '',
```

Comment:
@engahmeds3ed Simplified the fixture a bit, we know the capability we want to check, so we can mock only the returned value. We also can use the `restored` key to return the expected value